.travis.yml

```diff
@@ -160,7 +160,11 @@ script:
         fi;
         USE_DIRS="";
         for DIR in ${MNE_DIRS}; do
-          USE_DIRS="mne/${DIR}/tests ${USE_DIRS}";
+          if [ "${DIR}" == "." ]; then
+            USE_DIRS="mne/tests ${USE_DIRS}";
```

Comment:
I guess for completeness we might want `mne/*.py` to also catch doc tests on `mne/*.py` files